boa/src/syntax/ast/op.rs

```diff
@@ -40,15 +40,16 @@ impl Display for NumOp {
                 NumOp::Sub => "-",
                 NumOp::Div => "/",
                 NumOp::Mul => "*",
-                NumOp::Pow => "**",
+                NumOp::Exp => "**",
                 NumOp::Mod => "%",
             }
         )
     }
 }
 
 #[derive(Clone, Debug, Trace, Finalize, PartialEq)]
-/// A unary operation on a single value
+/// A unary operation on a single value   
+/// <https://tc39.es/ecma262/#prod-UnaryExpression>
```

Comment:
In this particular case, I would leave the title as-is, and add the link with the explanation just below:
```suggestion
/// A unary operation on a single value
///
/// For more information, please check: <https://tc39.es/ecma262/#prod-UnaryExpression>
```